lib/classes/board.rb

```diff
@@ -24,18 +24,7 @@ def game_over
 		def formatted_grid
 			grid.each_with_index do |row, outer|
 				r = row.each_with_index.map do |col, inner|
-					if col.value.empty?
-						case outer
-						when 0
-							inner + 1
-						when 1
-							inner + 4
-						when 2
-							inner + 7
-						end
-					else
-						col.value
-					end
+					col.value.empty? ? inner +  1 + (3 * outer) : col.value
```

Comment:
Can we store `row.length` in a variable and use it instead of the `3` to avoid [magic numbers?](https://www.quora.com/What-is-a-magic-number-in-programming)

Reply:
Thanks will do that now. Resolved with commit  [acf4f8bba97343d448e80321113fee525ea50369](https://github.com/adriaanbd/tic-tac-toe/pull/18/commits/acf4f8bba97343d448e80321113fee525ea50369)